dattri/algorithm/tracin.py

```diff
@@ -48,9 +47,9 @@ def f(params, image, label):
                     return loss(yhat, label)
                 ```.
                 This examples calculates the loss of the model on input data-label pair.
-            params_list (Union[List[dict], List[str]]): The parameters of the target
-                function. The input should be a list of dictionaries, where the keys
-                indicate the name of a parameter and the value is the parameter tensor.
+            model (torch.nn.Module): The PyTorch model to be attibuted.
+            params_list (List[str]): The parameters of the model, should be a list of
```

Comment:
The name `params_list` does not align well with its content now

Reply:
will make it `checkpoint_list`.